string relay
---
bool boardExists
bool foundRelay
int8  relayNumber
